Loop/Managers/LoopDataManager.swift

```diff
@@ -40,6 +40,19 @@ final class LoopDataManager {
             NotificationCenter.default.removeObserver(observer)
         }
     }
+    
+    // Integral restrospective correction parameters
+    private let currentDiscrepancyGain = 1.0 // Standard retrospective correction gain
```

Comment:
IRC algorithm state should be moved into its own class.

Reply:
Done that. Would be great if you could take a look and let me know if anything else needs be done. Then, I'll look into unit tests and issue report, will likely have questions about how to proceed.

Reply:
Looking good. The class should be moved into its own file, and that will allow room for some documentation about how it works. Being an important core algorithm, it would be good to spend some time documenting the general functionality of IRC, the functions in the class, their parameters, and any static parameters.  https://nshipster.com/swift-documentation/